css/main.css

```diff
@@ -458,6 +461,100 @@ body {
   margin: 18px 0;
 }
 
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
+
+.modal__header {
+  justify-content: space-between;
+}
+
+.modal__header img {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
+.modal__image {
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.modal__dsc__second {
+  flex-direction: column;
+  align-items: flex-start;
+  flex-basis: 40%;
+}
+
+.modal__dsc__second div {
+  margin: 1rem 0;
+}
+
+.modal__dsc__second p {
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 0.5rem;
+}
+
+.modal__btn {
+  margin: 1rem 0;
+}
```

Comment:
- [ ] For mobile screens please aligned the model buttons in the center and make sure it matches the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167877442-cf561834-e25f-481f-99a6-a8125bf1a940.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167877491-aab2f0f6-4cfd-4583-abb4-f308efa1ede3.png)